common/internal_model/src/main/scala/weco/catalogue/internal_model/identifiers/IdentifierType.scala

```diff
@@ -26,6 +26,11 @@ object IdentifierType extends Enum[IdentifierType] {
   def apply(id: String): IdentifierType =
     IdentifierType.withName(id)
 
+  case object Tei extends IdentifierType {
+    val id = "tei-manuscript-id"
+    val label = "Tei manuscript id"
```

Comment:
👍 

Do we need another type of identifier for the multiple works inside a TEI file, as you said at standup?

Reply:
Not for now, but possibly once I implement Adrian's suggestion for representing those multiple works. But all of that is still in modelling so I wouldn't worry for now